todo_app/data/todo_items.py

```diff
@@ -0,0 +1,26 @@
+
+class Item:
+
+    def __init__(self, id, short_id, name, desc, due, status = 'Not Started'):
+        self.id = id
+        self.short_id = short_id
+        self.name = name
+        self.desc = desc
+        self.due = due
+        self.status = status
+
+    @classmethod
+    def fromTrelloCard(cls, card, list):
+        due_date_string = str(card['due'])
+        due = due_date_string[0:10]
```

Comment:
On your comment about `strptime` - it can be quite fiddly! And this is made more complex if you assume the date is optional, which I think your code does. One way, based on [the format mentioned in this SO answer](https://stackoverflow.com/questions/127803/how-do-i-parse-an-iso-8601-formatted-date) would be:
```python
    @classmethod
    def fromTrelloCard(cls, card, list):
        due_string = ''
        if card['due'] is not None:
            # format here should match the Trello datetimes
            trello_date_format = '%Y-%m-%dT%H:%M:%S.%fZ'
            # use strptime to convert the string to a datetime
            due_datetime = datetime.strptime(card['due'], trello_date_format)
           # convert to just the date, and then take the string
            due_string = str(due_datetime.date())
      
        return cls(card['id'],card['idShort'], card['name'], card['desc'], due_string, list['name'])
```